lib/cosmos/topics/command_topic.rb

```diff
@@ -21,6 +21,9 @@
 
 module Cosmos
   class CommandTopic < Topic
+    COMMAND_ACK_TIMEOUT_MS = 5000
+    COMMAND_ACK_RETRY_MS = 250 # How much time to wait between ACK checks
```

Comment:
Not sure how configurable these need to be? Waiting 5s for a command ack seems to be plenty.

Reply:
Five seconds is fine except for some command/response cases.   For those cases it really needs to be the write_timeout given to the interface.  Not sure how to get that here though

Reply:
Me neither ... marked as TODO